src/action.cpp

```diff
@@ -1043,6 +1043,8 @@ cata::optional<tripoint> choose_adjacent_highlight( const std::string &message,
     }
     if( highlighted ) {
         wrefresh( g->w_terrain );
+        // prevent hiding panels when examining an object
```

Comment:
This comment is no longer accurate, and I'm fairly certain there's no need to redraw here either.

Reply:
You're right. I removed a couple of other provably-unnecessary draw_panels calls elsewhere, this is demonstrably useless too since it can't occlude the sidebar